### PR TITLE
Add xei, oku and dragonswap trades

### PR DESCRIPTION
**Pull request by Hosuke**
#6474 

### Update!
Please build spells in the proper [subproject](../dbt_subprojects/) directory. For more information, please see the main [readme](../README.md), which also links to a GH discussion with the option to ask questions.

### Contribution type
Please check the type of contribution this pull request is for:

- [ ] New spell(s)
- [x] Adding to existing spell lineage
- [ ] Bug fix

**Note:** You can safely discard any section below which doesn't apply based on selection above

---

### For new spell(s)
If you are building new spell(s), please provide the following information:

- **Spell name(s):** *schema.table_name*
- **Description:** [Detailed description of the new spell(s) and their purpose]
- **Who are the new spell(s) for?** [Internal team or general community]
- **How will the new spell(s) be used downstream?** [Description of downstream usage]
- **Implementation details:** [Information on how the spell(s) are implemented]
- **Test instructions:** [How to test the new spell(s)]
- **Related issue(s):** [Link to related issues, if any]

---

### For adding to existing spell lineage
If you are adding to an existing spell lineage, please provide the following information:

- **Description:** [Description of the changes made]

---

### For bug fixes
If you are fixing a bug, please provide the following information:

- **Description:** [Description of the bug fix]
- **Steps to reproduce:** [How to reproduce the bug]
- **Implementation details:** [Information on how the bug was fixed]
- **Test instructions:** [How to test the fix]
- **Related issue(s):** [Link to related issues, if any]

---

### Additional information
Please provide any additional information that might help us review your pull request:

- [Any additional information]

---

Thank you for your contribution!

### CODE REVIEW

File: dbt_subprojects/dex/models/trades/sei/_schema.yml [hunk 1]
```diff
@@ -22,4 +22,38 @@ models:
...
+    config:
+      tags: [ 'sei', 'dex', 'trades', 'dragon_swap', 'v2' ]
+    description: "DragonSwap sei base trades"
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - tx_hash
+            - evt_index
+#      - check_dex_base_trades_seed:
+#          seed_file: ref('dragon_swap_sei_base_trades_seed')
```
Comment (Hosuke) [inline dbt_subprojects/dex/models/trades/sei/_schema.yml:59]:
```suggestion
      - check_dex_base_trades_seed:
          seed_file: ref('dragon_swap_sei_base_trades_seed')
```

File: sources/_sector/dex/trades/sei/_sources.yml
Comment (jeff-dude) [inline sources/_sector/dex/trades/sei/_sources.yml:1]: building a new file here makes me wonder where we store sources for `jelly_swap` 🤔 
can we move those sources here to this new file?

**Review by jeff-dude — CHANGES_REQUESTED** [review]

File: sources/_sector/dex/trades/sei/_sources.yml
Comment (Hosuke) [reply sources/_sector/dex/trades/sei/_sources.yml:1]: I am also having this question. (Potentially in the labels sources)
I will try to move the `jelly_swap` related sources here.

**Review by jeff-dude — APPROVED** [review]
great stuff, thanks again for the speed here!

**Pull request merged**